servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/SniCompleteChannelSingle.java

```diff
@@ -55,6 +55,7 @@ public void handlerAdded(final ChannelHandlerContext ctx) throws Exception {
             // Force a read to get the SSL handshake started. We initialize pipeline before
             // SslHandshakeCompletionEvent will complete, therefore, no data will be propagated before we finish
             // initialization.
+            ctx.pipeline().fireUserEventTriggered(PipelineInitializedEvent.INSTANCE);
```

Comment:
The above comment is related to `ctx.read()`, consider moving this event before the comment (same for all other places).